exercises/finaleworkflow/README.md

```diff
@@ -0,0 +1,49 @@
+# Exercise 4: Finale Workflow
+
+One of the benefits of Temporal is that it provides SDKs for several
+languages and you can use multiple languages in the context of a single
+Workflow. For example, you might write your Workflow in TypeScript but use
+Java or Go for an Activity in that workflow.
+
+The last exercise in this workshop does exactly that. The Workflow
+itself is written in TypeScript, but the Activity that is executed as part
+of this Workflow is written in Java, as is the Worker that runs it.
+Since the Activity is written in Java, it's able to use a Java graphics
+library that would otherwise be would be incompatible with a typical
+TypeScript program.
+
+# Start the Worker (Java)
+
+In one terminal, run the following command from the `finaleworkflow` directory:
+
+```
+java -classpath \
+java-activity-and-worker.jar \
+io.temporal.training.PdfCertWorker
+```
+
+# Edit the Workflow (dotnet)
+
+Open the [Program.cs](./src/Client/Program.cs) file, and change the argument passed to the Workflow from 'Maxim Fateev' to your name.
+
+# Start the Workflow (dotnet)
+
+Run the command below from the src directory in another terminal to start the Workflow:
```

Comment:
Minimally, `src` should be in monospaced here:

```suggestion
Run the command below from the `src` directory in another terminal to start the Workflow:
```

However, I think it would be better still to add a new exercise step asking the learner to change into the `src` directory. I overlooked that in the prose, so having it as a command in an exercise step will make it more obvious.